contrib/lib/src/serve.rs

```diff
@@ -314,6 +315,15 @@ impl StaticFiles {
         self.rank = rank;
         self
     }
+
+    /// Sets the default content type
+    ///
+    /// If a file does not have an extension, this value will be used
+    /// for the content-type header
```

Comment:
Let's also change this to be more inline with my suggestions for the `NamedFile::with_default()` docstring.